data/language/nl-NL.yml

```
@@ -2191,3 +2191,209 @@ strings:
+  2253: "Show cash popups"
+  2254: "{SMALLFONT}{COLOUR BLACK}Toggle visibility of cash popups from construction and vehicles on/off"
+  2255: "Start server"
+  2256: "Sluit server"
+  2257: "Verbinding verbreken"
+  2258: "{COLOUR WINDOW_2}Lengte: {COLOUR BLACK}{INT32_2DP} tegel(s), {INT32} wagons"
+  2259: "{COLOUR WINDOW_2}Lengte: {COLOUR BLACK}{INT32_2DP} tegel(s)"
+  2260: "{COLOUR WINDOW_2} Lengte: {COLOUR BLACK}{INT32_2DP} tegel(s)"
+  2261: "{NEWLINE}{COLOUR WINDOW_2} Lengte: {COLOUR BLACK}{INT32_2DP} tegel(s)"
+  2262: "{NEWLINE}Lengte:  {INT32_2DP} tegel(s)"
+  2263: "{COLOUR WINDOW_2}{UINT16}/{UINT16}"
+  2264: "Reuzenscreenshot"
+  2265: "{COLOUR WINDOW_2} Bestandsnaam: {COLOUR BLACK}{STRINGID}"
+  2266: "Scenario-opties"
+  2267: "Objectselectie"
+  2268: "Gameplaytweaks"
+  2269: "Eigenaarsvoorkeuren"
+  2270: "Autosave-voorkeuren"
+  2271: "Stadsuitbreiding uitschakelen"
+  2272: "{SMALLFONT}{COLOUR BLACK}Wanneer ingeschakeld kunnen steden zich niet vernieuwen of uitbreiden over tijd."
+  2273: "Complete scenario challenge"
+  2274: "Wis"
+  2275: "Fout bij laden van objecten"
+  2276: "{COLOUR BLACK}Het bestand kon niet worden geladen omdat sommige objecten waar het naar verwijst ontbreken of beschadigd zijn. Een lijst van deze objecten staat hieronder."
+  2277: "{COLOUR BLACK}Object-ID"
+  2278: "{COLOUR BLACK}Type"
+  2279: "{COLOUR BLACK}Checksum"
+  2280: "Speel geluidseffecten voor nieuwe berichten"
+  2281: "{SMALLFONT}{COLOUR BLACK}Als dit ingeschakeld is, zullen nieuwe berichten vergezeld worden van een juichende of joelende menigte."
+  2282: "{SMALLFONT}{COLOUR BLACK}Maak omgekeerde ordertabel"
+  2283: "Naam en type"
+  2284: "HO"
+  2285: "HB"
+  2286: "R"
+  2287: "S"
+  2288: "{SMALLFONT}{COLOUR BLACK}Objectnaam en type van het tegelelement"
+  2289: "{SMALLFONT}{COLOUR BLACK}Basishoogte van het tegelelement"
+  2290: "{SMALLFONT}{COLOUR BLACK}Spelingshoogte van het tegelelement"
+  2291: "{SMALLFONT}{COLOUR BLACK}Richting van het tegelelement"
+  2292: "{SMALLFONT}{COLOUR BLACK}Spookstatus van het tegelelement"
+  2293: "Beginner"
+  2294: "Advanced"
+  2295: "Expert"
+  2296: "Vanilla"
+  2297: "Overig"
+  2298: "Componenten"
+  2299: "Sorteer op jaar van ontwerp"
+  2300: "Sorteer op naam"
+  2301: "Niet-aangedreven"
+  2302: "Aangedreven"
+  2303: "Onvergrendeld"
+  2304: "Vergrendeld"
+  2305: "Vracht"
+  2306: "Alle vrachttypes"
+  2307: "{MOVE_X 10}{STRINGID} {SPRITE}"
+  2308: "»{MOVE_X 10}{STRINGID} {SPRITE}"
+  2309: "{COLOUR BLACK}{STRINGID} {SPRITE}"
+  2310: "Vrachtloos"
+  2311: "Schakel laadpenalties voor voertuigen uit"
+  2312: "{SMALLFONT}{COLOUR BLACK}Wanneer ingeschakeld zullen de penalties rond laadtijden worden uitgeschakeld voor voertuigen die zich in te korte stations bevinden"
+  2313: "Doorzichtige gebouwen"
+  2314: "Doorzichtige wegen"
+  2315: "Doorzichtige decoratie"
+  2316: "Doorzichtige sporen"
+  2317: "Doorzichtige bomen"
+  2318: "Doorzichtige gebouwen togglen"
```

```suggestion
  2318: "Doorzichtige gebouwen aan/uit"
```
(Hetzelfde geldt natuurlijk voor de rest.)